### PR TITLE
[TIMOB-25005] Remove Ti.UI.iPhone namespace

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25005

### CODE REVIEW

File: apidoc/Titanium/UI/iPhone/ListViewSeparatorStyle.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ extends: Titanium.Proxy
 since: "3.2.0"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: "5.2.0"
+    removed: "7.0.0"
+    notes: Use <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE> and <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE> instead.
```
Comment (vijaysingh-axway) [inline apidoc/Titanium/UI/iPhone/ListViewSeparatorStyle.yml]: Rather "Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE" it should be "Titanium.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE". Same with "Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE".

File: apidoc/Titanium/UI/iPhone/TableViewSeparatorStyle.yml [hunk 2]
```diff
@@ -7,6 +7,10 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: "5.2.0"
+    removed: "7.0.0"
+    notes: Use <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE> and <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE> instead.
```
Comment (vijaysingh-axway) [inline apidoc/Titanium/UI/iPhone/TableViewSeparatorStyle.yml]: Same as above .

File: apidoc/Titanium/Media/Media.yml [hunk 3]
```diff
@@ -944,6 +944,7 @@ properties:
     osver: {ios: {min: "7.0"}}
     deprecated:
         since: "5.2.0"
+        since: "7.0.0"
```
Comment (vijaysingh-axway) [inline apidoc/Titanium/Media/Media.yml]: Change "since" to "remove" .

**Comment by hansemannn** [comment]
@vijaysingh-axway Great catch! Updated the PR.

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by mukherjee2** [comment]
@hansemannn can you please resolve the conflicts that are reported above?

**Comment by hansemannn** [comment]
@mukherjee2 Done.

**Review by mukherjee2 — APPROVED** [review]
Node Version: 6.11.5
NPM Version: 3.10.10
Mac OS: 10.13
Appc CLI: 7.0.0-master.6
Appc CLI NPM: 4.2.11-2
Titanium SDK version: 7.0.0 locally built on PR/9239
Appcelerator Studio vers 4.10.0
Xcode 9.1/iOS 11.1 iPhone 7 Plus
This ticket is validated. I tried some properties that were moved to Ti.UI.iOS, and I tried a property that was moved to Ti.UI, and apps built and worked fine for both. FR passed.

**Comment by mukherjee2** [comment]
@hansemannn there seems to be more conflicts now since the last resolution. Can you please merge after resolving again? It seems like others are working on the same files, and there will be conflicts again if the merge takes too long. FR has already passed now.

**Comment by mukherjee2** [comment]
@hansemannn @vijaysingh-axway can you please resolve conflicts and merge?

**Comment by mukherjee2** [comment]
@eric34 Can you please merge this PR?

**Pull request merged**